### DISCUSSION TITLE
Import Private Key

### DISCUSSION THREAD

**Discussion by cenon-delrosario** [gujjwal00/avnc#172] [General]
I am trying to import a private key for using SSH but it keeps saying "Invalid private key file"

**Comment by gujjwal00** [comment]
Hi @skeezmoe, after it says "Invalid private key file", please go to `Tools => Logs` in AVNC Settings, copy logs and paste here.

**Comment by cenon-delrosario** [comment]
Hi Gaurav, thank you for the support! I found the issue. I originally generated the rsa key from the app called JuiceSSH and exported the private key there. I have since generated the key from the remote server and it imports that with no issues. Thank you for coming back so soon thou :)
